
Fix compilation for recent 4.3 bdb versions ...

  - Rene Rebe <rene@exactcode.de>

--- libgda-1.3.1-orig/providers/bdb/gda-bdb-recordset.c	2005-01-10 13:29:38.000000000 +0100
+++ libgda-1.3.1/providers/bdb/gda-bdb-recordset.c	2005-04-06 09:39:36.667078360 +0200
@@ -228,7 +228,7 @@
 	/* get the number of records in the database */
 	ret = dbp->stat (dbp,
 			 &statp,
-#if BDB_VERSION < 40000
+#if BDB_VERSION < 40000 ||  BDB_VERSION > 40300
 			 NULL,
 #endif
 			 0);
